wcfsetup/install/files/lib/data/user/UserAction.class.php

```diff
@@ -325,6 +326,13 @@ public function update() {
 					}
 				}
 			}
+			
+			if (isset($this->parameters['data']['password'])) {
```

Comment:
Use `array_key_exists`. `null` is a valid value for the `Invalid` password algorithm and `isset` will return false for a `null` value.